### ISSUE TITLE
Script to rewrite addonUrls stored in DB

### ISSUE THREAD

**Issue by sciurus** [mozilla/normandy#1597]
I've realized the full addonUrl is stored in Normandy's database. The values there refer to an S3 bucket. For example,

```
normandy=> select arguments_json from public.recipes_reciperevision where name = 'T12Clone';
                                                                                               arguments_json                                                                        
                        
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
------------------------
 {"name": "T12OptStudy", "description": "T12OptDescription", "addonUrl": "https://net-mozaws-stage-us-east-1-normandy.s3.amazonaws.com/extensions/vk_scrobbler-2.1.1.1-fx.xpi", "isEn
rollmentPaused": false}
(1 row)
```

After the GCP migration, we'll want to rewrite them to refer to the domain we'll use to front our GCS bucket so that we can retire the S3 bucket. To do this I think a python script that 

1. finds all the recipe revisions with addonURL in their arumgent_json
1. and updates them to replace the domain with a new value (either provided as an argument to the script or read from whatever setting we implement for #1567)

would be handy.

**Comment by sciurus** [comment]
I've realized that since we didn't do #1567 , this script is not sufficient. That's because it only changes the domain, but if we are going straight to the storage bucket the path needs to change as well.

For example

https://net-mozaws-prod-us-west-2-normandy.s3.amazonaws.com/extensions/side-view-control@shield.mozilla.org-0.1.3-signed_jQeRzcx.xpi becomes https://storage.googleapis.com/moz-fx-normandy-prod-addons/extensions/side-view-control@shield.mozilla.org-0.1.3-signed_jQeRzcx.xpi

Old domain: net-mozaws-prod-us-west-2-normandy.s3.amazonaws.com
New domain: storage.googleapis.com

Old path excluding filename: /extensions/
New path excluding filename: /moz-fx-normandy-prod-addons/extensions/

**Comment by jaredkerim** [comment]
@sciurus Is this still necessary or will this be obviated by the transition away from managing addons in normandy towards a new addon system?

**Comment by sciurus** [comment]
This should have been fixed by #1703 . I have not verified it yet, but will when I do https://bugzilla.mozilla.org/show_bug.cgi?id=1525648 which we can talk about tomorrow.
